pom.xml

```diff
@@ -45,6 +98,50 @@
         </repository>
     </repositories>
 
+    <!-- conditional build of jar with dependencies. To get it, run mvn command with -P fatjar -->
+    <profiles>
+        <profile>
+            <id>javac-release</id>
+            <activation>
+                <jdk>[9,)</jdk>
+            </activation>
+            <properties>
+                <maven.compiler.release>8</maven.compiler.release>
+            </properties>
+        </profile>
```

Comment:
Got that solution from [here](https://issues.apache.org/jira/browse/MCOMPILER-339)